### PR TITLE
Fixed PebbleSync slopeOrdinal().

### PR DESCRIPTION
**Pull request by jstevensog**
Now uses BgReading.slopeName like most other slope arrow displays.
Pebble will now show same slope arrows as all other xDrip displays.
Note: NO ACTUAL CHANGE TO BgGraphBuilder.  Not sure why git thought it was
modified.

### CODE REVIEW

**Comment by StephenBlackWasAlreadyTaken** [comment]
If this were in a non master branch on xdrip experimental I could resolve the conflicts and merge but I cant without jumping through a bunch of hoops since the branch is officially on your repo

**Comment by jstevensog** [comment]
That's weird. It was done against xDrip-E master, which I had simply pulled into my repo. Also, I already have it in xDrip-E. Must be an old PR. 

---- Stephen Black wrote ----

**Comment by StephenBlackWasAlreadyTaken** [comment]
github is always fun!

File: app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgGraphBuilder.java [hunk 1]
```diff
@@ -61,7 +62,8 @@ public BgGraphBuilder(Context context){
         this.prefs = PreferenceManager.getDefaultSharedPreferences(context);
         this.highMark = Double.parseDouble(prefs.getString("highValue", "170"));
         this.lowMark = Double.parseDouble(prefs.getString("lowValue", "70"));
-        this.doMgdl = (prefs.getString("units", "mgdl").compareTo("mgdl") == 0);
+        //this.doMgdl = (prefs.getString("units", "mgdl").compareTo("mgdl") == 0);
+        this.doMgdl = (prefs.getString("units", "mgdl").compareTo("mmol") != 0);
```
Comment (mgranberry) [inline app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgGraphBuilder.java]: Codacy Issue found: [Useless parentheses.](https://www.codacy.com/app/matthias-granberry/xDrip-Experimental/file/1312212526/issues/source?bid=2735053&fileBranchId=2735108#l66)

Comment (mgranberry) [inline app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgGraphBuilder.java]: Codacy Issue found: [Useless parentheses.](https://www.codacy.com/app/matthias-granberry/xDrip-Experimental/file/1312212526/issues/source?bid=2735053&fileBranchId=2735108#l66)

File: app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/PebbleSync.java [hunk 2]
```diff
@@ -171,27 +171,16 @@ public int getBatteryLevel() {
     }
 
     public String slopeOrdinal(){
```
Comment (mgranberry) [inline app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/PebbleSync.java:166]: Codacy Issue found: [The method slopeOrdinal() has an NPath complexity of 256](https://www.codacy.com/app/matthias-granberry/xDrip-Experimental/file/1312212540/issues/source?bid=2735053&fileBranchId=2735108#l173)

Comment (mgranberry) [inline app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/PebbleSync.java:166]: Codacy Issue found: [The method slopeOrdinal() has an NPath complexity of 256](https://www.codacy.com/app/matthias-granberry/xDrip-Experimental/file/1312212540/issues/source?bid=2735053&fileBranchId=2735108#l173)

**Pull request closed without merging**